app/views/header.js

```diff
@@ -0,0 +1,36 @@
+import Mn from 'backbone.marionette';
+import application from '../application';
+import Tracks from '../collections/tracks';
+
+const Header = Mn.ItemView.extend({
+
+    template: require('./templates/header'),
+
+    events: {
+        'click #reset-upnext': 'resetUpNext',
+        'click #delete-playlist': 'deletePlaylist'
+    },
+
+    //modelEvents: { change: 'render' }, // DON'T WORK
```

Comment:
Only works if the model is passed at initialization. You should want to instanciate the `HeaderView` in _content.js_ with `this.showChildView('header', new HeaderView({model: application.headerInfos))` which is a cleaner solution.